crates/cheatnet/tests/contracts/target/dev/cheatnet_testing_contracts.starknet_artifacts.json

```diff
@@ -0,0 +1 @@
+{"version":1,"contracts":[{"id":"lldvf1koh10qu","package_name":"cheatnet_testing_contracts","contract_name":"HelloStarknet","artifacts":{"sierra":"cheatnet_testing_contracts_HelloStarknet.sierra.json","casm":"cheatnet_testing_contracts_HelloStarknet.casm.json"}},{"id":"r22odu2r2tgug","package_name":"cheatnet_testing_contracts","contract_name":"PrankChecker","artifacts":{"sierra":"cheatnet_testing_contracts_PrankChecker.sierra.json","casm":"cheatnet_testing_contracts_PrankChecker.casm.json"}}]}
```

Comment:
Do we wan't to commit compiled contracts artifacts? It will be quite impractical to maintain.

Reply:
done